TestsFoundation/TestsFoundation/TestsFoundation.h

```diff
@@ -18,6 +18,12 @@
 
 #import <UIKit/UIKit.h>
 
+#include <XCTest/XCTest.h>
+@interface XCUIElementQuery ()
+- (NSArray<XCUIElementSnapshot> *) allMatchingSnapshotsWithError:(NSError **)err;
```

Comment:
What is this?

Reply:
This is how "exists" works internally. It's an undocumented API, but it's the best way I can find to restore the pre-xcode-11 behavior.